alea/model.py

```diff
@@ -95,19 +95,16 @@ def __init__(
         self._confidence_interval_kind = confidence_interval_kind
         self.confidence_interval_threshold = confidence_interval_threshold
         self.asymptotic_dof = asymptotic_dof
-        nominal_values = kwargs.get("nominal_values", {})
-        self._define_parameters(parameter_definition, nominal_values)
+        self._define_parameters(parameter_definition)
 
         self._check_ll_and_generate_data_signature()
+        self.set_nominal_values(overwrite_static=True, **kwargs.get("nominal_values", {}))
```

Comment:
@hammannr Can we keep `set_nominal_values ` in the future so that `_define_parameters` only handle the definition of `Parameters`? And for the `need_reinit` parameters, I think they can be defined correctly just by `set_nominal_values`.